### ISSUE TITLE
icon title not relevant

### ISSUE THREAD

**Issue by atomGit** [MarkaPola/Linkificator#13]
if you move the toolbar icon to one of the panels, the icon title is not relevant - the title currently tells the number of requests processed, but not the add-name

to follow a more standard convention, please consider prefixing the text with "Linkificator" and perhaps the processed text string can be replaced with an icon badge (number of links processed), while the processed text info would go in a tool-tip on mouse-over

**Comment by MarkaPola** [comment]
I agree with you, the behavior is not satisfactory. Unfortunately, I cannot change this because this is controlled by WebExtension framework.

Currently, when the button is put in some panel, what is displayed is the tooltip, not the title of the button. This is, from my point of view, a bug of WebExtension framework.

The only way to enhance, on my side, a little the current situation is to add a customization to activate/deactivate tooltip of the button...
